### PR TITLE
*: notices about importing 1.0 cluster with TiUP

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What is changed, added or deleted? (Required)

- add a notice for DM-portal when importing 1.0 cluster with TiUP
- refer automatically upgrade in manually upgrade guide

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@GMHDBJD @TomShawn PTAL

File: zh/maintain-dm-using-tiup.md [hunk 1]
```diff
@@ -241,6 +241,10 @@ tiup dm patch prod-cluster /tmp/dm--hotfix.tar.gz -N 172.16.4.5:8261
 
 ## 导入 DM-Ansible 部署的 DM 1.0 集群并升级
 
+> **注意：**
+>
+> TiUP 不支持导入 1.0 集群中的 DM-portal 组件。
```
Comment (TomShawn) [inline zh/maintain-dm-using-tiup.md:246]:
```suggestion
> TiUP 不支持导入 1.0 集群中的 DM Portal 组件。
```

File: en/manually-upgrade-dm-1.0-to-2.0.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ summary: Learn how to manually upgrade TiDB data migration from v1.0.x to v2.0.x
 
 This document introduces how to manually upgrade the TiDB DM tool from v1.0.x to v2.0.x. The main idea is to use the global checkpoint information in v1.0.x to start a new data replication task in the v2.0.x cluster.
 
+For how to automatically upgrade the TiDB DM tool from v1.0.x to v2.0.x, please refer to [Using TiUP to automatically import the 1.0 cluster deployed by DM-Ansible](maintain-dm-using-tiup.md#import-and-upgrade-a-dm-10-cluster-deployed-using-dm-ansible).
```
Comment (TomShawn) [inline en/manually-upgrade-dm-1.0-to-2.0.md:10]:
```suggestion
For how to automatically upgrade the TiDB DM tool from v1.0.x to v2.0.x, refer to [Using TiUP to automatically import the 1.0 cluster deployed by DM-Ansible](maintain-dm-using-tiup.md#import-and-upgrade-a-dm-10-cluster-deployed-using-dm-ansible).
```

File: en/maintain-dm-using-tiup.md [hunk 3]
```diff
@@ -245,6 +245,10 @@ tiup dm patch prod-cluster /tmp/dm--hotfix.tar.gz -N 172.16.4.5:8261
 
 ## Import and upgrade a DM 1.0 cluster deployed using DM-Ansible
 
+> **Note:**
+>
+> TiUP doesn't support to import the DM-portal component in the 1.0 cluster.
```
Comment (TomShawn) [inline en/maintain-dm-using-tiup.md:250]:
```suggestion
> TiUP does not support importing the DM Portal component in the 1.0 cluster.
```

File: en/maintain-dm-using-tiup.md [hunk 4]
```diff
@@ -245,6 +245,10 @@ tiup dm patch prod-cluster /tmp/dm--hotfix.tar.gz -N 172.16.4.5:8261
 
 ## Import and upgrade a DM 1.0 cluster deployed using DM-Ansible
 
+> **Note:**
+>
+> TiUP does not support importing the DM Portal component in the 1.0 cluster.
```
Comment (yikeke) [inline en/maintain-dm-using-tiup.md:250]:
```suggestion
> TiUP does not support importing the DM Portal component in a DM 1.0 cluster.
```

**Pull request merged**